app/gui2/src/components/GraphEditor/GraphNodeComment.vue

```diff
@@ -16,19 +16,17 @@ const { documentation: astDocumentation } = useAstDocumentation(
   graphStore,
   () => props.node.outerExpr,
 )
-const documentation = computed({
-  // This returns the same value as the `astDocumentation` getter, but with fewer reactive dependencies.
-  get: () => props.node.documentation ?? '',
-  set: (value) => (astDocumentation.value = value),
-})
+// This returns the same value as `astDocumentation.state`, but with fewer reactive dependencies.
+const documentation = computed(() => props.node.documentation ?? '')
 
 syncRef(editing, useFocusDelayed(textEditor).focused)
 </script>
 <template>
   <div v-if="editing || props.node.documentation?.trimStart()" class="GraphNodeComment">
     <PlainTextEditor
       ref="textEditor"
-      v-model="documentation"
+      :modelValue="documentation"
+      @update:modelValue="astDocumentation.set"
```

Comment:
Perhaps a matter of taste, but I preferred the old solution here. Or maybe it has some drawbacks?

Reply:
It relied on a side effect in a computed setter; [Paweł suggested](https://github.com/enso-org/enso/pull/10064#pullrequestreview-2077298561) replacing that pattern with a state getter and updating separately.